dll/win32/msgina/lang/it-IT.rc

```diff
@@ -181,12 +181,18 @@ BEGIN
     DEFPUSHBUTTON "OK", IDOK, 115, 179, 50, 14
 END
 
-IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 275, 146
+// Luna XP Style Dialog
+IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 275, 135
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "Luna-XP-style Shut Down dialog"
+CAPTION "Spegnimento di ReactOS"
```

Comment:
Please don't copy-paste localized strings from other sources. Let the native translators deal with the translations.

Reply:
for @reactos/lang-indonesian language rc, i approve it.